ui-tests/src/test/java/org/openmrs/reference/MarkPatientDeceasedTest.java

```diff
@@ -0,0 +1,50 @@
+package org.openmrs.reference;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.openmrs.reference.groups.BuildTests;
+import org.openmrs.reference.page.AdministrationPage;
+import org.openmrs.reference.page.FindPatientPage;
+import org.openmrs.reference.page.MarkPatientDeceasedPage;
+import org.openmrs.uitestframework.test.TestData;
+
+public class MarkPatientDeceasedTest extends ReferenceApplicationTestBase {
+	
+	private TestData.PatientInfo patient;
```

Comment:
Thanks @insookwa  for the great work, i think we have same issue with identations at some point, so please fix your identations to fit openmrs identation formatter, then make your methods look more cleaner